examples/platforms/simulation/openthread-core-simulation-config.h

```diff
@@ -124,8 +124,10 @@
 #endif
 
 #ifndef OPENTHREAD_CONFIG_LOG_PREPEND_UPTIME
+#if !OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE
 #define OPENTHREAD_CONFIG_LOG_PREPEND_UPTIME 1
 #endif
+#endif
```

Comment:
This is good to add but suggest the following instead:

```suggestion
#ifndef OPENTHREAD_CONFIG_LOG_PREPEND_UPTIME
#define OPENTHREAD_CONFIG_LOG_PREPEND_UPTIME ! OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE
#endif
```